src/pages/AppDetail.svelte

```diff
@@ -1,19 +1,47 @@
 <script>
   import { onMount } from "svelte";
+  import Pill from "../components/Pill.svelte";
   import { fetchJSON } from "../state/api";
 
   export let params;
   const URL = `data/${params.app}/index.json`;
   let app;
+  let pillColors = { bgColor: "#2c5282", borderColor: "#ffffff" };
```

Comment:
There is no need to make this a variable (since it's always the same), you can just hardcode the values when creating the component.

Reply:
hey @wlach I need clarity on this one. Do you mean i should  set the color as I am calling the component like this
`<Pill data={app} pillBgColor ="#2c5282" />`

Reply:
Yup that's correct. No need to name it `pillBgColor` though, just `bgColor` should be sufficient.

Reply:
Done